e2e/dapp-connections/WalletConnect.yaml

```diff
@@ -0,0 +1,79 @@
+appId: ${APP_ID}
+tags:
+  - browser
+  - parallel
+---
+- clearState
+- launchApp:
+    arguments:
+      isE2ETest: true
+- runFlow: ../utils/Prepare.yaml
+- runFlow: ../utils/ImportWalletWithKey.yaml
+
+# Navigate to browser
+- swipe:
```

Comment:
I think it is better to use

```
- tapOn:
    id: 'tab-bar-icon-DappBrowserScreen'
```

To navigate to the browser, we already test that swiping works in Discover. I've seen a few cases were using swipe was more flaky and this will also avoir loading the extra discover tab (My latest PR makes tab loading lazy when in test).

Reply:
good callout! fixed